src/main/groovy/org/grails/plugins/web/Java8GrailsPlugin.groovy

```diff
@@ -26,6 +26,27 @@ import java.lang.reflect.Field
 
 class Java8GrailsPlugin extends Plugin {
 
+    public static final String DEFAULT_JSR310_OFFSET_DATE_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ssZ"
+    public static final String DEFAULT_JSR310_OFFSET_TIME_FORMAT = "HH:mm:ssZ"
+    public static final String DEFAULT_JSR310_LOCAL_DATE_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ss"
+    public static final String DEFAULT_JSR310_LOCAL_DATE_FORMAT = "yyyy-MM-dd"
+    public static final String DEFAULT_JSR310_LOCAL_TIME_FORMAT = "HH:mm:ss"
+    public static final String DEFAULT_JSR310_ZONED_DATE_TIME_FORMAT = "yyyy-MM-dd'T'HH:mm:ssZ"
+
+    /**
+     * Default JSR310 Date formats. Used when no custom formats are supplied.
+     *
+     * Note that the JSR310 formats extend the defaults provided in {@link DataBindingGrailsPlugin#DEFAULT_DATE_FORMATS}
+     */
+    public static final List<String> DEFAULT_JSR310_FORMATS = [
```

Comment:
Since the format for Zoned and OffsetDateTime are the same, perhaps this should be a set?

Reply:
Yep - good call.

If I make this a `Set` do you think its worth going the extra mile to change everything through to (and including) `Jsr310ConvertersConfiguration#formatStrings` to also be the same? It potentially saves a few cycles from misconfiguration....

Reply:
I made it a `Set` all the way down